ui/src/dataExplorer/components/SaveAsTaskForm.tsx

```diff
@@ -109,14 +103,58 @@ class SaveAsTaskForm extends PureComponent<Props & WithRouterProps> {
           onSubmit={this.handleSubmit}
           canSubmit={this.isFormValid}
           dismiss={dismiss}
-          tokens={tokens}
+          tokens={this.getRelevantTokens()}
           selectedToken={selectedToken}
           onTokenChange={this.handleTokenChange}
         />
       </SpinnerContainer>
     )
   }
 
+  private getRelevantTokens() {
```

Comment:
if you make this a getter `private get relevantTokens()` you wont need to evaluate the function on line 106